src/vue/modules/issuances-explorer/store/index.spec.js

```diff
@@ -0,0 +1,95 @@
+import { mutations, getters, actions } from './index'
+import { types } from './types'
+
+import { Issuance } from '../wrappers/issuance'
+
+import { Sdk } from '@/sdk'
+import { OP_TYPES } from '@tokend/js-sdk'
+
+describe('issuances-explorer.module', () => {
+  const accountId = 'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ'
+  const issuances = [{
+    id: '1',
+    amount: '100.000000',
+  }, {
+    id: '2',
+    amount: '50.000000',
+  }]
+
+  describe('mutations', () => {
+    let state
+
+    beforeEach(() => {
+      state = {
+        accountId: '',
+        issuances: [],
+      }
+    })
+
+    it('SET_ACCOUNT_ID should properly modify state', () => {
+      mutations[types.SET_ACCOUNT_ID](state, accountId)
+
+      expect(state).to.deep.equal({
+        accountId: accountId,
+        issuances: [],
```

Comment:
this test shouldn't know about `issuances` prop

Reply:
Updated